cli/cli_app.go

```diff
@@ -248,6 +252,32 @@ func runTerragrunt(terragruntOptions *options.TerragruntOptions) error {
 		}
 	}
 
+	if shouldPrintTerragruntInfo(terragruntOptions) {
+		type TerragruntInfoGroup struct {
```

Comment:
Could you move the type definition to a top-level item, rather than defining it within an if-statement? This will also allow re-using the type info at test time.

Reply:
done